### PR TITLE
Mark release 1.2.2

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-1.2.0...master
 
+## [1.2.2] - 2021-04-06
+[1.2.2]: https://github.com/atlassian-labs/db-replica/compare/release-1.2.0...release-1.2.2
```
Comment (dagguh) [inline CHANGELOG.md:27]: Huh, 1.2.1 doesn't exist neither in Maven repo nor in Git tags. Weird, but correct 😃

**Review by dagguh — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (wyrzyk) [reply CHANGELOG.md:27]: I'm not sure if I misused the release pipeline. I selected chose `Patch` and I thought it'll release 1.2.1.
Zrzut ekranu 2021-04-6 o 16 21 35

Comment (dagguh) [reply CHANGELOG.md:27]: Yep it's the `-Prelease.incrementer=incrementPatch`. I tested it locally with `-Prelease.localOnly`

Comment (afaruga-atlassian) [reply CHANGELOG.md:27]: Is it bug or feature?

Comment (dagguh) [reply CHANGELOG.md:27]: It's most likely a human error. Locally I tested various `markNextVersion` and `release` calls and I couldn't reproduce this.
Calling `markNextVersion` repeatedly does increment the version even if it is not released, but we shouldn't call it many times. Unless an GH Action was red and rerun.

Comment (wyrzyk) [reply CHANGELOG.md:27]: It was green without re-run. Let's wait for the next patch release to see if it'll behave the same way.

**Pull request merged**